ddr/test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+# Copyright (c) 2018, 2019 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+#############################################################################
+
+include(OmrDDRSupport)
+
+add_executable(ddrgentest
+    test_subdir/testFileInSubDir.cpp
+    sample1.cpp
+    sample2.cpp
+    sample3.cpp
+    sample4.c
+    test.cpp
+)
+
+make_ddr_set(testset)
+target_enable_ddr(ddrgentest testset)
+
+file(GLOB_RECURSE headerfiles "${CMAKE_CURRENT_SOURCE_DIR}/*.h[pp]?")
```

Comment:
Normally `[]` means any of the characters in between: that's not what you want here.
Perhaps you meant to use parentheses:
> file(GLOB_RECURSE headerfiles "${CMAKE_CURRENT_SOURCE_DIR}/*.h(pp)?")